samples/simple_chat.py

```diff
@@ -0,0 +1,22 @@
+from autogen import UserProxyAgent, ConversableAgent, config_list_from_json
+
+
+def main():
+    # Load LLM inference endpoints from an env variable or a file
+    # See https://microsoft.github.io/autogen/docs/FAQ#set-your-api-endpoints
+    # and OAI_CONFIG_LIST_sample.
+    # For example, if you have created a OAI_CONFIG_LIST file in the current working directory, that file will be used.
+    config_list = config_list_from_json(env_or_file="OAI_CONFIG_LIST")
+
+    # Create the agent that uses the LLM.
+    assistant = ConversableAgent("agent", llm_config={"config_list": config_list})
+
+    # Create the agent that represents the user in the conversation.
+    user_proxy = UserProxyAgent("user")
```

Comment:
```suggestion
    user_proxy = UserProxyAgent("user", code_execution_config={"work_dir": "coding"})
```
In case code execution is required, we specify the working directory here.

Reply:
Including code_execution_config and the coding directory would make this example significantly more complex for beginners, because the installation section of the README stresses that docker should be used for code execution, and many users haven't used docker. By leaving out the parameter, this sample makes the point that coding is not always required.

Reply:
@sonichi , @skzhang1 has added the working directory as you suggested above. I would still recommend not doing this, but it's not a big issue. If you still prefer to include the working directory, then this conversation can be resolved and the PR can be merged. Otherwise, Dany can remove the working directory to make this script as simple as possible.

Reply:
@rickyloynd-microsoft Thanks for your suggestion! I was convinced. If we say "in ChatGPT style" in title, we should not specify the working directory since ChatGPT didn't support this feature.

Reply:
Removing code_execution_config doesn't disable code execution. To disable it, need to set it to False.